OvmfPkg/IntelTdx/Sec/SecMain.c

```diff
@@ -68,6 +68,11 @@ SecMtrrSetup (
     return;
   }
 
+  if (CcProbe () == CcGuestTypeIntelTdx) {

```

Comment:
please add detailed comment on why it is skipped.

Reply:
Thanks, updated now